snap/quota/resources.go

```diff
@@ -262,6 +314,20 @@ func (qr *Resources) ValidateChange(newLimits Resources) error {
 		}
 	}
 
+	// Verify journal limits not being removed
+	if qr.Journal != nil && newLimits.Journal != nil {
+		if qr.Journal.Size != nil && newLimits.Journal.Size != nil && newLimits.Journal.Size.Limit == 0 {
+			return fmt.Errorf("cannot remove journal size limit from quota group")
```

Comment:
We could still have `qr.Journal.Size.Limit` to be zero, right? (though this depends on how we proceed with the `omitempty` flag)

Reply:
I think you answered :-)